Assets/Plugins/AndroidEditor/Editor/PlatformSpecificConfigEditorAndroid.cs

```diff
@@ -69,6 +69,8 @@ public void OnGUI()
                 overrideValues = configFile.currentEOSConfig.overrideValues;
             }
 
+            EpicOnlineServicesConfigEditor.AssigningBoolField("Link EOS-SDK Statically", ref configFile.currentEOSConfig.linkLibStatically);
```

Comment:
This config option should be put in a plugin editor config section, similar to how `EOSPluginEditorToolsConfig` is done, as this config option is ideally for runtime only specific configuration.